woocommerce/plugin/wp-content/plugins/indodana-payment/indodana-payment-gateway.php

```diff
@@ -238,6 +238,21 @@ public function getTotalTaxAmount($cart)
     return (float) $cart->get_total_tax();
   }
 
+  public function getTotalAdminFeeAmount($cart)
+  {
+    return (float) 0;
+  }
+
+  public function getTotalAdditionalFeeAmount($cart)
+  {
+    return (float) $cart->get_fee_total();
+  }
+
+  public function getTotalInsuranceFeeAmount($cart)
+  {
+    return (float) 0;
```

Comment:
Why are we hardcoding this? is it because we just want to support it so it doesn't break the code?

Reply:
Primary reason is because Woocommerce didn't have insurance fee (at least to our understanding, no). You don't need to pass `insuranceFeeAmount` also though (because will be defaulted to 0 on `IndodanaCommon` layer for backward compatibility). But for better practice (so we're aware of it), it's better to just specify the `insuranceFeeAmount` equals to `0`

Reply:
I see, got it, can you please add it to the code comment so new engineers will get the reasoning?